doc/source/release-notes/Percona-Server-5.7.23-23.rst

```diff
@@ -97,4 +97,4 @@ Other Bugs Fixed
 * :psbug:`4561` \"Read after free at Binlog_crypt_data::load_latest_binlog_key()\"
 * :psbug:`4587` \"ROCKSDB_INCLUDE_RFR macro in wrong file\"
   
-.. |release| replace:: 5.7.23-23
+.. 5.7.23-23 replace:: 5.7.23-23
```

Comment:
Do we need this replacement? It replaces itself, and you haven't put it as a variable in the text. I think it can be safely removed

Reply:
The replacement is not printed and can be removed at a later date.